core/src/main/java/com/sopt/core/designsystem/component/button/NoostakButton.kt

```diff
@@ -0,0 +1,38 @@
+package com.sopt.core.designsystem.component.button
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.material3.Button
+import androidx.compose.ui.unit.dp
+import com.sopt.core.designsystem.theme.NoostakTheme
+
+@Composable
+fun NoostakButton(
+    text: String,
+    textColor: Color,
+    buttonColor: Color,
+    onButtonClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    isEnabled: Boolean = true
+) {
+    Button(
+        modifier = modifier,
```

Comment:
P2 : 하단 bottom btn이라면 아래와 같이 size를 지정하는게 더 좋을 것 같아요
```suggestion
        modifier = modifier.fillMaxWidth(),
```

또한, padding도 해당 컴포넌트 코드에서 입력해놓으면 재사용할 때 편리할 것 같아요! 하단 버튼은 재사용되는 뷰가 많아서 각자 컴포넌트를 생성했을거라(저 포함) 생각하는데 추후 하나로 합치는 것이 좋아보이네요